core/ir/x86/decode_table.c

```diff
@@ -8995,9 +8995,9 @@ const instr_info_t evex_Wb_extensions[][4] = {
     {OP_vpcompressq, 0x66388b48, catSIMD, "vpcompressq", We, xx, KEb, Ve, xx, mrm|evex|reqp|ttt1s, x, END_LIST},
     {INVALID, 0x66388b58, catUncategorized, "(bad)", xx,xx,xx,xx,xx,no,x,NA},
   }, { /* evex_W_ext 168 */
-    {OP_vpexpandd, 0x66388908, catSIMD, "vpexpandd", We, xx, KEw, Ve, xx, mrm|evex|reqp|ttt1s, x, END_LIST},
+    {OP_vpexpandd, 0x66388908, catSIMD, "vpexpandd", Ve, xx, KEw, We, xx, mrm|evex|reqp|ttt1s, x, END_LIST},
```

Comment:
This is worth listing in the changelist at https://github.com/DynamoRIO/dynamorio/blob/master/api/docs/release.dox#L138.  Please list this decoder bug fix change there.

Reply:
Done.